core/block/editor/clipboard/clipboard.go

```diff
@@ -585,6 +589,34 @@ func (cb *clipboard) newHTMLConverter(s *state.State) *html.HTML {
 	return html.NewHTMLConverter(cb.fileService, s, cb.fileObjectService)
 }
 
+func (cb *clipboard) processFileBlock(f *model.BlockContentOfFile) {
+	if cb.fileObjectService == nil {
```

Comment:
it have to be initialized at this moment

Reply:
Removed this check